docs/_posts/2017-09-08-520-release.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: blogpost
+title:  "5.2.0 Release"
+section_id: blog
+date:   2017-09-08 13:30:00
```

Comment:
Is the date auto-populated ? if not, we need to update it to reflect the date of posting.

Reply:
I'm not sure why it's not showing up for you, but the date has been updated to today's date.

Reply:
yeah. i see '2017-09-08'. But if you see today or tomorrow date (whenever we post the blog), this is fine then.